deploy/helm-chart/templates/deployment.yaml

```diff
@@ -58,38 +61,20 @@ spec:
             - DAC_OVERRIDE
```

Comment:
can we also double check the capabilities here? For example, in NIC we have https://github.com/nginxinc/kubernetes-ingress/blob/494fc891adc8600ca2d4e73a84cf68076c67d75a/deployments/deployment/nginx-ingress.yaml#L66-L69